spec.html

```diff
@@ -39728,6 +39914,26 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-typedarray-create-same-type" type="abstract operation">
+        <h1>
+          TypedArrayCreateSameType (
+            _exemplar_: a TypedArray,
+            _argumentList_: a List of ECMAScript language values,
+          ): either a normal completion containing a TypedArray or a throw completion
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It is used to specify the creation of a new TypedArray using a constructor function that is derived from _exemplar_. Unlike TypedArraySpeciesCreate, which can construct custom TypedArray subclasses through the use of @@species, this operation always uses one of the built-in TypedArray constructors.</dd>
+        </dl>
+        <emu-alg>
+          1. Let _constructor_ be the intrinsic object listed in column one of <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for _exemplar_.[[TypedArrayName]].
```

Comment:
We don't typically number the column names like this. Although verbose, we would typically say

```suggestion
          1. Let _constructor_ be the intrinsic object associated with the constructor name _exemplar_.[[TypedArrayName]] in <emu-xref href="#table-the-typedarray-constructors"></emu-xref>.
```

Separately, we should probably split that column into two at some point. I have no idea why they are combined. But even if we do, that wording should be fine.